src/edge/views.py

```diff
@@ -10,20 +11,39 @@
 
 from edge.models import Fragment, Genome, Operation
 from edge.io import IO
+from edge import import_gff
+from edge.tasks import build_genome_blastdb
 
 
 def genome_export(request, genome_id):
     get_genome_or_404(genome_id)
     io = IO(Genome.objects.get(pk=genome_id))
-
     response = HttpResponse(content_type='text/plain')
     response['Content-Disposition'] = "attachment; filename=\"g%s.gff\"" % genome_id
     io.to_gff_file(response)
     return response
 
 
+def genome_import(request):
```

Comment:
you should stick some REST validation around this, making sure it only gets called on `POST`s, etc; you can do this with the `@require_POST()` decorator or a class-based view that only implements the `post` method.

Reply:
Good thoughts! It's actually @require_http_methods(["POST"]) https://docs.djangoproject.com/en/1.11/topics/http/decorators/